src/backend/cdb/cdbpath.c

```diff
@@ -1566,6 +1551,21 @@ cdbpath_motion_for_join(PlannerInfo *root,
 	outer.bytes = outer.path->rows * outer.path->pathtarget->width;
 	inner.bytes = inner.path->rows * inner.path->pathtarget->width;
 
+	if (join_quals_contain_outer_references ||
+		CdbPathLocus_IsOuterQuery(outer.locus) ||
+		CdbPathLocus_IsOuterQuery(inner.locus))
+	{
+		/*
+		 * GPDB_FIXME: Consider Replicated locus.
```

Comment:
CBDB_FIXME?

Reply:
Forget to rename, and there are more than here, should be renamed too.

Reply:
Done.